package.json

```diff
@@ -111,18 +111,29 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/twilio-video": "^2.0.12",
     "@types/yup": "^0.29.3",
-    "@typescript-eslint/eslint-plugin": "^3.7.1",
-    "@typescript-eslint/parser": "^3.7.1",
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
     "cypress": "^4.11.0",
     "cypress-file-upload": "^4.0.7",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-config-prettier": "^6.15.0",
     "eslint-config-react-app": "^5.2.1",
+    "eslint-loader": "^3.0.3",
+    "eslint-plugin-flowtype": "^4.6.0",
```

Comment:
Do we need this? We don't use Flow. There may be some nice to have rules in there that are independent of it, but also wonder if that would conflict with prettier/etc things. Feels like adding extra risk at this stage, maybe can explore it as a separate PR in future if it makes sense to?

Docs mention needing to add `babel-eslint` added for this to work
- https://github.com/gajus/eslint-plugin-flowtype#installation